lib/controller/python/vehicle/driver.py

```diff
@@ -0,0 +1,230 @@
+# Copyright 1996-2022 Cyberbotics Ltd.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from controller import Supervisor
+import ctypes
+import os
+import sys
+
+
+class Driver(Supervisor):
+    INDICATOR_OFF = 0
+    INDICATOR_RIGHT = 1
+    INDICATOR_LEFT = 2
+    SPEED = 0
+    TORQUE = 1
+    # wiper modes
+    DOWN = 0
+    SLOW = 1
+    NORMAL = 2
+    FAST = 3
+
+    def __init__(self):
+        super().__init__()
+        if sys.platform == 'linux' or sys.platform == 'linux2':
+            path = os.path.join('lib', 'controller')
+            car = 'libcar.so'
+            driver = 'libdriver.so'
+        elif sys.platform == 'win32':
+            path = os.path.join('lib', 'controller')
+            car = 'car.dll'
+            driver = 'driver.dll'
+        elif sys.platform == 'darwin':
+            path = os.path.join('Contents', 'MacOS', 'lib', 'controller')
+            car = 'libcar.dylib'
+            driver = 'libdriver.dylib'
+        ctypes.cdll.LoadLibrary(os.path.join(os.environ['WEBOTS_HOME'], path, car))
+        self.api = ctypes.cdll.LoadLibrary(os.path.join(os.environ['WEBOTS_HOME'], path, driver))
+        self.api.wbu_driver_get_brake_intensity.restype = ctypes.c_double
```

Comment:
```suggestion
        self.api.wbu_driver_get_brake_intensity.restype = ctypes.c_double
        self.api.wbu_driver_get_current_speed.restype = ctypes.c_double
```

The following snippet of code:
```
while self.step() != -1:
  print(self.getCurrentSpeed())
```
produces incorrect large int values instead of floats without this additional line.